### PR TITLE
Dev branch milestone.

### PR DESCRIPTION
**Pull request by Nowembabazi**
**##In this milestone I achieved the following;**
-The Space Travelers' Hub consists of Rockets, Missions, and the My Profile section, and this PR shows the intersection of 
 user stories (requirements) with our joint implementations:

 **The Rockets section displays a list of all available SpaceX rockets.**
-The Missions section displays a list of current missions along with their brief description and participation status (i.e. users can 
 toggle between whether or not they are joining the missions)
-The My Profile section displays all reserved rockets and space missions.
-Use Redux: Fetch data and update Redux store
-Render UI:lists with useSelecto
-Redux: Write actions and reducers for booking rockets/dragons and joining missions                                                                   
-Redux: Write actions and reducers for canceling rockets/dragons and leaving missions
-Render UI: conditional components rendering
-Render UI: My Profile section

### CODE REVIEW

File: src/components/MissionProfiler.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <div>
+      <h1 className="head-text">My Missions</h1>
+      <table className="tour_list">
+        <tbody className="tour_body">
+          {
+            tourists.length < 1 ? (
+              <tr>
+                <td className="tour-now tour">No Mission Yet...</td>
+                <td><Link to="/missions">Join a Mission</Link></td>
```
Comment (prolajumokeoni) [inline src/components/MissionProfiler.js:17]: - [ ] Good job 👏🏿 adding a placeholder message when the "My Missions" or "My Rockets" lists are empty

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella  and @Nowembabazi 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors 
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️

- [ ] Kindly write unit tests with React Testing Library and Jest (for unit tests).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 💪 
- No linter errors
- correct flow is used
- good improvement based on the previous reviewer's suggestions
- good PR

 ## Required Changes ♻️

- [ ] As requested by the previous reviewer, please write unit tests and other tests for your components using React Testing Library and Jest 😃 👍 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewers.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**